### ISSUE TITLE
TaxBrain file-upload page needs updating

### ISSUE THREAD

**Issue by martinholmer** [OpenSourcePolicyCenter/PolicyBrain#500]
The [TaxBrain file-upload page](http://www.ospc.org/taxbrain/file/) needs an update to reflect the fact that there are two types of files: policy reform files and economic assumption files.

First, can we deal with a long-standing bug?  How about simply removing the broken link on the left (below the "Get Started" link) that is labeled "Input Form"?  Is that not an easy thing to do?  If for some reason it is not easy to remove it, then change it to "Input Forms" and make the link work by having it do exactly the same thing as the "Get Started" link does.

Second, I suggest something like this for the main page:
```
Reform File: See examples here

Select a policy reform file:         <---- add "policy" to correspond to "economic" below

<Choose File button>

Assumptions File: See examples here  <---- add label with here link undefined at the moment

Select an economic assumptions file:

<Choose File button>
```

@MattHJensen @brittainhard

**Comment by martinholmer** [comment]
In addition to the file-upload page enhancements in TB issue #500, there is an needed enhancement  to the results page generated by uploaded file(s).  Now the output page starts this way:
```
THE FOLLOWING REFORM FILES WERE UPLOADED FOR THIS SIMULATION:
```
And then it reproduces the uploaded policy reform file and uploaded economic assumptions file.
See this [output page](http://www.ospc.org/taxbrain/10230/) for an example.

Since Matt requested splitting policy reforms and economic assumptions into two separate types of JSON files, the heading above is confusing.  When somebody gets to this page they already know about the difference between reform and assumption files, so I suggest the top label be clarified by dropping the word `REFORM` so that it reads:
```
THE FOLLOWING FILE(S) WERE UPLOADED FOR THIS SIMULATION:
```
Adding the parentheses around the `S` in `FILES` makes it clear that in many instances (when the default economic assumptions are being used) only one file is uploaded.

@MattHJensen @brittainhard

**Comment by MattHJensen** [comment]
Summary

- [ ] On results page, "THE FOLLOWING REFORM FILES WERE UPLOADED FOR THIS SIMULATION:" should be "THE FOLLOWING FILE(S) WERE UPLOADED FOR THIS SIMULATION:"
- [ ] On taxbrain/file, remove "Input Form" from sidebar nav. 
- [ ] On taxbrain/file, change the main text/layout to:

```
Reform File: See examples here

Select a policy reform file:         <---- add "policy" to correspond to "economic" below

<Choose File button>

Assumptions File: See examples here  <---- add label with here link undefined at the moment

Select an economic assumptions file:

<Choose File button>
```

**Comment by martinholmer** [comment]
**This is an update to TaxBrain issue #500, which was first raised on March 4, 2017.**

After completing the user documentation, the TaxBrain file-upload page show look like this:
```
Reform File: See examples [here]<link to taxcalc/reforms/REFORMS.md>

Select a policy reform file: 

<Choose File button>

Assumption File: See examples [here]<link to taxcalc/responses/ASSUMPTIONS.md>

Select a response assumption file:

<Choose File button>
```

**Comment by martinholmer** [comment]
@MattHJensen,  Where do we stand on the simple enhancements requested in issue #500?

**Comment by martinholmer** [comment]
@MattHJensen said on April 22, 2017 in issue #500 about the file-upload-results page:
> On results page, "THE FOLLOWING REFORM FILES WERE UPLOADED FOR THIS SIMULATION:" should be "THE FOLLOWING FILE(S) WERE UPLOADED FOR THIS SIMULATION:"

Despite the many improvements included in TaxBrain 1.0.3, this simple change has not been made.
Is there any chance, this change can be made in the next version of TaxBrain?

When making this change, can we also have the uploaded file(s) be shown **unchanged** from the way they were when uploaded?  TaxBrain 1.0.3 is removing all leading spaces on each line, so these uploaded files:
```
// r1.json assumes reform with the following provisions:
// - adhoc raises in OASDI maximum taxable earnings in 2018, 2019 and 2020,
//     with _SS_Earnings_c wage indexed in subsequent years
// - raise personal exemption amount _II_em in 2018, keep it unchanged for
//     two years and then resume its price indexing in subsequent years
// - implement a 20% investment income AGI exclusion beginning in 2019
{
    "policy": {
        "_SS_Earnings_c": {"2018": [400000],
                           "2019": [500000],
                           "2020": [600000]},
        "_II_em": {"2018": [8000]},
        "_II_em_cpi": {"2018": false,
                       "2020": true},
        "_ALD_InvInc_ec_rt": {"2019": [0.20]}
    }
}

// a3.json assumes some customized analysis assumptions, which means:
// - no consumption response when computing marginal tax rates
// - some behavioral substitution and income response
// - no differences in baseline growth assumptions
// - no growth response to reform
{
    "consumption": {
    },
    "behavior": {
        "_BE_sub": {"2013": [0.3]},
        "_BE_inc": {"2013": [-0.1]}
    },
    "growdiff_baseline": {
    },
    "growdiff_response": {
    }
}
```
look like this on the TaxBrain results page:

![screen shot 2017-10-03 at 3 30 37 pm](https://user-images.githubusercontent.com/12170745/31146043-665a788c-a853-11e7-851b-228fa22b0168.png)


@hdoupe @GoFroggyRun

**Comment by martinholmer** [comment]
@GoFroggyRun,  Can you fix the remaining problems in PolicyBrain issue #500 as discussed [here](https://github.com/OpenSourcePolicyCenter/PolicyBrain/issues/500#issuecomment-333961358)?

The title change should be quite easy.

The loss in spacing when the uploaded file(s) are displayed is probably easy to fix as well.
The spacing is lost probably because the file content is being shown as HTML, which ignores the spacing in the file(s).  Could the fix be as simple as surrounding the text of the file(s) with `<verbatim> ... </verbatim>` tags?

@MattHJensen @hdoupe
